docs/source/user-guide/installation.md

```diff
@@ -26,9 +26,14 @@ Make sure the following requirements are met and software installed on your mach
 - Linux
 - Apple OSX (Intel and Apple Silicon)
 
+## Supported Apache Spark versions
+- 3.3
+- 3.4
+- 3.5
+
```

Comment:
are we fully supporting 4.0? I can see the build maven profile for 4.0

Reply:
No not fully supported yet